sat/cli/bootsys/state_recorder.py

```diff
@@ -321,20 +321,24 @@ def do_state_capture(args):
     Raises:
         SystemExit: if there is a failure to capture state.
     """
-    state_recorders = [PodStateRecorder()]
-
-    failed = []
-    LOGGER.info('Capturing system state.')
-    with BeginEndLogger('system state capture'):
-        for sr in state_recorders:
-            try:
-                LOGGER.info(f'Capturing {sr.description}')
-                sr.dump_state()
-            except StateError as err:
-                LOGGER.error(f'Failed to capture {sr.description}: {err}')
-                failed.append(sr)
-
-    if failed:
+    try:
+        state_recorders = [PodStateRecorder()]
+    except Exception as err:
```

Comment:
It's probably best to catch the specific type of Exception we are looking for, in this case, `S3ResourceCreationError`.

```suggestion
    except S3ResourceCreationError as err:
```